src/plugins/so_management/server/services/management.ts

```diff
@@ -17,19 +17,13 @@
  * under the License.
  */
 
-import { SavedObject } from '../types';
-import { ISavedObjectTypeRegistry } from '../saved_objects_type_registry';
+import { ISavedObjectTypeRegistry, SavedObject } from 'src/core/server';
+
+export type ISavedObjectsManagement = PublicMethodsOf<SavedObjectsManagement>;
```

Comment:
As no one is going to instantiate `SavedObjectsManagement` outside this plugin, you could export only the type and name it `SavedObjectsManagement`.